index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+  <script src="https://use.fontawesome.com/92bb8db824.js"></script>
+  <title>Portfolio Sneak Peak</title>
+</head>
+<body>
+  <div>
+    <header>
+      <img class="status-bar" src="images/status-bar.png" alt="Portfolio Status Bar">
+      <nav>
+        <div class="logo">
+          <img src="images/app-bar.png" alt="Logo image">
+          <i class="fa fa-bars"></i>
+        </div>
+      </nav>
+    </header>
+    <section>
+      <h1>I’m Sylvester,
+          Glad to see you!
+      </h1>
+      <p class="intro">
+        I’m a software developer! I can help you build a product, 
+        feature or website. Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <h3>LET'S CONNECT</h3>
```

Comment:
- [ ] As suggested, please let's make sure that we follow the given design. The navbar on your portfolio is coming slightly after the image which I believe should be the other way round. Kindly fix this. As you can see in the screenshot the navbar is a bit above the image.
![image](https://user-images.githubusercontent.com/68167810/120699484-1a0d3880-c4b9-11eb-8d41-0806d7a653ea.png)

- [ ] Please make sure that (Lets Connect) is a `button` or an `a` tag as guided by the previous reviewer this is to ensure that your code looks more semantic and organized.
[OPTIONAL] You can also consider making the` h1 ` tag a bit bolder as from the given design.

Reply:
Please take a look at the code again and advise on the exact required changes to be made. I think I just one review left.

Reply:
Hello Sly-Cloud if you have made any changes on your code then you can request for another review. However if the review I made was not clear please point out the exact place so that i may clarify to you more.